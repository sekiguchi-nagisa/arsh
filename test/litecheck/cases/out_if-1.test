#!/bin/sh

ls -la
# CHECK_IF: @@@@@
echo @@@@@
echo AAAAA