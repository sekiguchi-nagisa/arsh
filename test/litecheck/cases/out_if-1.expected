CHECK_IF on line 4,
@@ -1 +1,2 @@
 @@@@@
+AAAAA
stdout is mismatched